### PR TITLE
Add Web Vitals Experience docs

### PR DESCRIPTION
**Pull request by gr0**
This PR adds web vitals docs for Experience. They look as follows:

webvitals

Any comments and adjustments welcome :)

### CODE REVIEW

File: docs/experience/webvitals.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+title: Web Vitals
+description: Gathering information about web vitals - essential site health metrics
+
+Supported by Chrome, the [Web Vitals](https://web.dev/vitals/) is a Google initiative to provide unified guidance for metrics that are crucial for delivering the best user experience for the web-based applications.
```
Comment (amir-hadzic) [inline docs/experience/webvitals.md:4]: >for the web-based application

I would suggest "for websites and webapps". I think we used those terms in other docs pages.

File: docs/experience/webvitals.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  src="../../images/experience/webvitals/webvitals.png"
+  title="Web Vitals Overview"
+/>
+
+### Core Web Vitals
+
+Both [Sematext Experience](/experience/) and [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals) support the Core Web Vitals out of the box and include the following metrics:
+
+ * **Largest Contentful Paint (LCP)** - measures *page loading* performance. Google suggests that **LCP** should be **below 2500 milliseconds**, which means that to provide good user experience, the largest contentful paint should occur within 2.5 seconds of when the page first starts loading.
+ * **First Input Delay (FID)** - measures *interactivity* of the web application. Google suggests that to provide good user experience, pages should have a first input delay of **less than 100 milliseconds**.
```
Comment (amir-hadzic) [inline docs/experience/webvitals.md:18]: FID is not supported by Synthetics, maybe add a small note?

File: docs/experience/webvitals.md (hunk 1 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Done.

File: docs/experience/webvitals.md (hunk 2 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Good idea, added.

File: docs/experience/webvitals.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  alt="Web Vitals Overview"
+  src="../../images/experience/webvitals/webvitals.png"
+  title="Web Vitals Overview"
+/>
+
+### Core Web Vitals
+
+Both [Sematext Experience](/experience/) and [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals) support the Core Web Vitals out of the box and include the following metrics:
+
+ * **Largest Contentful Paint (LCP)** - measures *page loading* performance. Google suggests that **LCP** should be **below 2500 milliseconds**, which means that to provide good user experience, the largest contentful paint should occur within 2.5 seconds of when the page first starts loading.
```
Comment (otisg) [inline docs/experience/webvitals.md:17]: If we write these definitions as sections with anchors then we can easily link to them, say from PU, or from blog posts, or from SC itself.

File: docs/experience/webvitals.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ * **First Input Delay (FID)** - measures *interactivity* of the web application. Google suggests that to provide good user experience, pages should have a first input delay of **less than 100 milliseconds**. Please note that FID is not measured in [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals).
+ * **Cumulative Layout Shift (CLS)** - measures *visual* stability of the web page. To provide a good user experience Google suggests that the pages should maintain a cumulative layout shift of **less than 0.1**.
+
+### Other Web Vitals Metrics
+
+In addition to the Core Web Vitals, both [Sematext Experience](/experience/) and [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals) support the supplemental Web Vitals metrics often used to help in diagnostics of specific issues on the web page. Those metrics are:
```
Comment (otisg) [inline docs/experience/webvitals.md:23]: to help in => to help with

File: docs/experience/webvitals.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+ * **Time to First Byte (TTFB)** - measures the time at which the server sends a response for the request. It is suggested that to provide a great user experience the time to the first byte should be **below 600 milliseconds**. Higher values may discourage users from interacting with the site and are usually cause application performance issues such as long page loads.
```
Comment (otisg) [inline docs/experience/webvitals.md:25]: > the time at which
Is it really the time AT?  That would imply a specific time, like 16:32:01
Maybe it's "the time in takes the server to send..."?

File: docs/experience/webvitals.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  src="../../images/experience/webvitals/webvitals.png"
+  title="Web Vitals Overview"
+/>
+
+### Core Web Vitals
+
+Both [Sematext Experience](/experience/) and [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals) support the Core Web Vitals out of the box and include the following metrics:
+
+ * **Largest Contentful Paint (LCP)** - measures *page loading* performance. Google suggests that **LCP** should be **below 2500 milliseconds**, which means that to provide good user experience, the largest contentful paint should occur within 2.5 seconds of when the page first starts loading.
+ * **First Input Delay (FID)** - measures *interactivity* of the web application. Google suggests that to provide good user experience, pages should have a first input delay of **less than 100 milliseconds**. Please note that FID is not measured in [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals).
```
Comment (otisg) [inline docs/experience/webvitals.md:18]: provide good => provide a good

File: docs/experience/webvitals.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+ * **Time to First Byte (TTFB)** - measures the time at which the server sends a response for the request. It is suggested that to provide a great user experience the time to the first byte should be **below 600 milliseconds**. Higher values may discourage users from interacting with the site and are usually cause application performance issues such as long page loads.
+ * **First Contentful Paint (FCP)** - measures the time from when the page starts loading to when any part of the page's content is rendered on the screen. Unlike the largest contentful paint, the first contentful paint tells the site owner when the initial page element is rendered on the user screen. It is said that the first contentful paint should be kept **below 1000 milliseconds** to achieve a good user experience.
+
+### Web Vitals in Experience
+
+The information about Web Vitals in Experience is present as a set of numbers and graphs.
```
Comment (otisg) [inline docs/experience/webvitals.md:30]: numbers and graphs => numeric components and charts

File: docs/experience/webvitals.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+ * **Time to First Byte (TTFB)** - measures the time at which the server sends a response for the request. It is suggested that to provide a great user experience the time to the first byte should be **below 600 milliseconds**. Higher values may discourage users from interacting with the site and are usually cause application performance issues such as long page loads.
+ * **First Contentful Paint (FCP)** - measures the time from when the page starts loading to when any part of the page's content is rendered on the screen. Unlike the largest contentful paint, the first contentful paint tells the site owner when the initial page element is rendered on the user screen. It is said that the first contentful paint should be kept **below 1000 milliseconds** to achieve a good user experience.
+
+### Web Vitals in Experience
+
+The information about Web Vitals in Experience is present as a set of numbers and graphs. 
+
+The main panel of the Web Vitals presents information about the 75th percentile of both the Core and Other Web Vitals:
```
Comment (otisg) [inline docs/experience/webvitals.md:32]: The main panel of the Web Vitals => The main Web Vitals panel

The main panel of the Web Vitals presents information about the 75th percentile of both the Core and Other Web Vitals:  
==>
The main Web Vitals panel  shows the 75th percentile for both the Core and Other Web Vitals:

File: docs/experience/webvitals.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+The main panel of the Web Vitals presents information about the 75th percentile of both the Core and Other Web Vitals:  
+
+<img
+  class="content-modal-image"
+  alt="Core Web Vitals Panel"
+  src="../../images/experience/webvitals/webvitals_corepanel.png"
+  title="Core Web Vitals Panel"
+/>
+
+In addition to metrics panel, the information is presented in an easy to read graphs that visualize the data for the selected time period:
```
Comment (otisg) [inline docs/experience/webvitals.md:41]: In addition to metrics panel => In addition to the metrics panel
(btw. what is "metrics panel"?  Do we use that elsewhere?)

File: docs/experience/webvitals.md (hunk 4 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Done, adjusted, thanks.

File: docs/experience/webvitals.md (hunk 5 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Done, adjusted, thanks.

File: docs/experience/webvitals.md (hunk 7 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Done, adjusted, thanks.

File: docs/experience/webvitals.md (hunk 8 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Done, adjusted, thanks.

File: docs/experience/webvitals.md (hunk 6 above)
Comment (gr0) [reply docs/experience/webvitals.md:18]: Done, adjusted, thanks.

File: docs/experience/webvitals.md (hunk 9 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Adjusted and rephrased slightly.

File: docs/experience/webvitals.md [hunk 10]
```diff
@@ -38,7 +38,7 @@ The main panel of the Web Vitals presents information about the 75th percentile
   title="Core Web Vitals Panel"
 />
 
-In addition to metrics panel, the information is presented in an easy to read graphs that visualize the data for the selected time period:
+In addition to the panel presenting the 75th percentile, the information is presented in an easy to read graphs that visualize the data for the selected time period:
```
Comment (otisg) [inline docs/experience/webvitals.md:41]: the panel presenting ==> the panel showing
(avoid using "present.." twice in the same sentence)

File: docs/experience/webvitals.md (hunk 3 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Good idea, done.

File: docs/experience/webvitals.md (hunk 10 above)
Comment (gr0) [reply docs/experience/webvitals.md]: Done :)

**Comment by gr0** [comment]
Thanks for review guys!

**Pull request merged**